### PR TITLE
Remove GT asm

### PR DESCRIPTION
**Pull request by Glease**
depends on https://github.com/GTNewHorizons/GT5-Unofficial/pull/1339

### CODE REVIEW

**Review by mitchej123 — APPROVED** [review]
Will this need a dependencies version bump?

**Review by mitchej123 — DISMISSED** [review]
bump gt5u version

File: dependencies.gradle [hunk 1]
```diff
@@ -1,20 +1,20 @@
 dependencies {
-    compile('com.github.GTNewHorizons:GT5-Unofficial:5.09.41.12:dev')
-    compile('com.github.GTNewHorizons:StructureLib:1.1.6:dev')
-    compile('com.github.GTNewHorizons:NotEnoughItems:2.2.20-GTNH:dev')
-    compile('com.github.GTNewHorizons:CodeChickenCore:1.1.4:dev')
+    compile('com.github.GTNewHorizons:GT5-Unofficial:5.09.41.26:dev')
```
Comment (mitchej123) [inline dependencies.gradle]: 27

Comment (Dream-Master) [reply dependencies.gradle]: ok sec

**Comment by Dream-Master** [comment]
@Glease this and this pr https://github.com/GTNewHorizons/GT5-Unofficial/pull/1339 need to be reworked. Coil blocks not visible in inventories anymore.

**Pull request merged**